tests/conf/trainer.yaml

```diff
@@ -0,0 +1,26 @@
+trainer:
```

Comment:
@adamjstewart Here's an example of recursive instantiation using hydra. This lets you define callbacks, loggers, etc inside the config itself and not having to edit the actual `train.py` script to change these.